docs/running-headscale-linux.md

```diff
@@ -36,21 +36,27 @@ useradd \
 	--home-dir /var/lib/headscale/ \
 	--system \
 	--user-group \
-	--shell /usr/bin/nologin \
+	--shell /usr/sbin/nologin \
```

Comment:
Is this true for alle distros? which one is the reference?

Reply:
I assume for many distributions. This is certainly the case for debian based ones. According to this post for some more too. https://superuser.com/questions/830656/debian-linux-usr-bin-nologin-missing

Reply:
would it be better to add `$(which nologin)` instead to be compatible across distros?

Reply:
You are right, that does not work but `su -s /bin/bash - headscale` still does with nologin.